src/views/Gallery.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+    <div id="gallery-content">
+        <div id="header-mobile">
+            <div>
+                <h2>Specimen gallery</h2>
+                <h3>Self similarity telescope</h3>
+            </div>
+            <div type="button" id="change-button">
+                <span
+                    alt="Swap icon"
+                    id="change-icon"
+                    class="material-icons-sharp"
+                    >swap_horiz</span
+                >
+                <p id="change-text">Change Visualizer</p>
+            </div>
+        </div>
+
+        <h1 id="header">Specimen gallery</h1>
+
+        <div type="button" class="visualizer-bar">
+            <h2>Self similarity telescope</h2>
+            <span class="material-icons-sharp">keyboard_arrow_up</span>
+        </div>
+        <div class="gallery">
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+        </div>
+
+        <div type="button" class="visualizer-bar">
+            <h2>Self similarity telescope</h2>
+            <span class="material-icons-sharp">keyboard_arrow_up</span>
+        </div>
+        <div class="gallery">
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+        </div>
+    </div>
+</template>
+
+<script setup lang="ts">
+    import SpecimenCard from '../components/SpecimenCard.vue'
+
+    const sequenceName = '2-acid val of Z'
+    const cardName = 'Pinstripe'
+</script>
+
+<style lang="scss" scoped>
```

Comment:
Also, when I check out this branch, do `npm install`, and then `npm run dev` and visit the root page in a browser, I see
```
[plugin:vite:css] Preprocessor dependency "sass" not found. Did you install it? Try `npm install -D sass`.
```
I do not see `sass` mentioned in package.json. I suspect you need to do the npm install suggested and commit the resulting package and package-lock files. Of course, if it's the case `scss` (and hence `sass`) isn't actually used yet, per the other comment at this point, then you can remove the lang designation and this comment will be moot for now.

Reply:
This error was indeed caused by a missing element in package.json. I've made the appropriate changes.